### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by wyjzz** [Microsoft/CNTK#285]
I create a SimpleNetwork.The config file is as fellow.But when I run it,the layers size is not what I defined in config file.I use the simpleid sample,just change datafile.

configparameters: Simple.cntk:Simple_Demo_Train=[
    action = "train"
    SimpleNetworkBuilder = [
        layerSizes = 22:50*2:3
        trainingCriterion = "CrossEntropyWithSoftmax"
        evalCriterion = "ErrorPrediction"
        layerTypes = "Sigmoid"
        initValueScale = 1.0
        applyMeanVarNorm = true
        uniformInit = true
        needPrior = true
    ]
 reader = [
        readerType = "UCIFastReader"
        file = "$DataDir$/stock1.txt"
        miniBatchMode = "partial"
        randomize = "auto"
        verbosity = 1  

```
    features = [
        dim = 22        # two-dimensional input data
        start = 1      # Start with first element on line
    ]

    labels = [
        start = 0      # Skip two elements
        dim = 1        # One label dimension
        labelDim = 3   # Two labels possible
        labelMappingFile = "$DataDir$/SimpleMapping.txt"
    ]
]
```

Validating network. 25 nodes to process in pass 1.

Validating --> labels = InputValue -> [2 x _]
Validating --> W2 = LearnableParameter -> [2 x 50]
Validating --> W1 = LearnableParameter -> [50 x 50]
Validating --> W0 = LearnableParameter -> [50 x 2]
Validating --> features = InputValue -> [2 x *]
Validating --> MeanOfFeatures = Mean(features[2 x *]) -> [2 x 1]
Validating --> InvStdOfFeatures = InvStdDev(features[2 x *]) -> [2 x 1]
Validating --> MVNormalizedFeatures = PerDimMeanVarNormalization(features[2 x *], MeanOfFeatures[2 x 1], InvStdOfFeatures[2 x 1]) -> [2 x *]
Validating --> W0_features = Times(W0[50 x 2], MVNormalizedFeatures[2 x _]) -> [50 x *]
Validating --> B0 = LearnableParameter -> [50 x 1]
Validating --> W0_features+B0 = Plus(W0_features[50 x *], B0[50 x 1]) -> [50 x 1 x *]
Validating --> H1 = Sigmoid(W0_features+B0[50 x 1 x _]) -> [50 x 1 x *]
Validating --> W1_H1 = Times(W1[50 x 50], H1[50 x 1 x _]) -> [50 x *]
Validating --> B1 = LearnableParameter -> [50 x 1]
Validating --> W1_H1+B1 = Plus(W1_H1[50 x *], B1[50 x 1]) -> [50 x 1 x *]
Validating --> H2 = Sigmoid(W1_H1+B1[50 x 1 x _]) -> [50 x 1 x *]
Validating --> W2_H1 = Times(W2[2 x 50], H2[50 x 1 x _]) -> [2 x *]
Validating --> B2 = LearnableParameter -> [2 x 1]
Validating --> HLast = Plus(W2_H1[2 x *], B2[2 x 1]) -> [2 x 1 x *]
Validating --> CrossEntropyWithSoftmax = CrossEntropyWithSoftmax(labels[2 x *], HLast[2 x 1 x *]) -> [1]
Validating --> EvalErrorPrediction = ErrorPrediction(labels[2 x *], HLast[2 x 1 x *]) -> [1]
Validating --> PosteriorProb = Softmax(HLast[2 x 1 x *]) -> [2 x 1 x *]
Validating --> Prior = Mean(labels[2 x *]) -> [2 x 1]
Validating --> LogOfPrior = Log(Prior[2 x 1]) -> [2 x 1]
Validating --> ScaledLogLikelihood = Minus(HLast[2 x 1 x *], LogOfPrior[2 x 1]) -> [2 x 1 x *]

**Comment by yzhang87** [comment]
Which part was inconsistent? plz make sure you remove the old model file when you create a new one.

**Comment by pkranen** [comment]
Closing this one since not sufficient information was given since a long time. Please open a new issue if the problem remains with the latest bits. Thanks.
